google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -109,16 +110,56 @@ def _send_findings_to_gcs(self, violations, gcs_path):
                     tmp_violations.name, gcs_upload_path)
         return
 
-    def _transform_for_api(self, violations):
+    def _transform_for_api(self, violations, source_id=None):
         """Transform forseti violations to findings for CSCC API.
 
         Args:
             violations (dict): Violations to be sent to CSCC as findings.
+            source_id (str): Unique ID assigned by CSCC, to the organization
+                that the violations are originating from.
 
         Returns:
             list: violations in findings format; each violation is a dict.
         """
         findings = []
+
+        # beta api
+        if source_id:
+            LOGGER.debug('Transforming findings with beta API. source_id: %s',
+                         source_id)
+            for violation in violations:
+                # CSCC can't accept the full hash, so this must be shortened.
+                finding_id = violation.get('violation_hash')[:32]
+                finding = {
+                    'name': '{0}/findings/{1}'.format(
```

Comment:
Any character limits on any of these fields?

Reply:
I believe that the character limits are actually bigger than on alpha api.  Let me follow-up with the CSCC team to be sure.